src/components/Molecules/Footer/Footer.js

```diff
@@ -10,19 +10,27 @@ import {
   InnerWrapper,
   FooterBranding,
   FooterCopyright,
-  SocialIconWrapper
+  SocialIconWrapper,
+  Brand
 } from './Footer.style';
 
 const Footer = ({ navItems, footerCopy, campaign, ...rest }) => {
+  // Concatenate campaign in one string
+  const campaignName = campaign
+    .split(' ')
```

Comment:
Can you create a new const campaignName and use .replace(/ +/g, "").join('').toLowerCase() instead? So we can read what's doing.